docs/events.md

```diff
@@ -429,6 +431,148 @@ croct.track('cartViewed', {
 ```
 </details>
 
+### cartModified
+
+This event records the user modified the shopping cart.
+
+You should track this event when a user modifies the shopping cart page or summary.
+
+#### Properties
+
+This event supports the following properties:
+
+| Property                              | Type     | Required | Constraints                       | Description                                                                                                                                                                                                                                                                                                                                                 
+|---------------------------------------|----------|----------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
The formating/indentation is completely broken. Please refer to other sections for examples of how you should format this table.

Reply:
Have you verified if there is really an indentation problem?

Reply:
Never mind, that's an issue with the Github UI